pyquil/gate_matrices.py

```diff
@@ -287,6 +287,33 @@ def bitphase_flip_operators(p):
     'bitphase_flip': bitphase_flip_operators,
 }
 
+SIC0 = np.array([1, 0])
```

Comment:
Add a reference so people can check?

I personally use this http://info.phys.unm.edu/~caves/reports/infopovm.pdf see Eqns. 22 and 23

Reply:
cool; I'll add that in